### ISSUE TITLE
Support insert_all/3 with source

### ISSUE THREAD

**Issue by rschooley** [Nebo15/ecto_mnesia#52]
https://hexdocs.pm/ecto/Ecto.Repo.html#c:insert_all/3

> It expects a schema (MyApp.User) or a source ("users") 

I only see tests and an implementation for passing a schema to insert_all/3 in ecto_mnesia.

**Comment by AndrewDryga** [comment]
Yes, we have noted in the readme that we don't support schemaless queries. I plan to work on adapter during next few weeks and going to take a look how to support them :).
